services/api/src/routes/categories.js

```diff
@@ -14,39 +15,25 @@ router
     validate({
       body: Joi.object({
         name: Joi.string(),
```

Comment:
name should be remove from here, the keyword search options can be used instead

Reply:
the `keywordFields` is part of `getSearchQuery`, not `searchValidation`... I thought about this as well, it would be nice to specify in one place instead of 3, however it's difficult to not split it up without doing something that's too restrictive.

At any rate this will be solved or at least alleviated I think when the schema to joi validation stuff lands so I'm planning to look at this then and see how these search utils fit in there